### DISCUSSION TITLE
MME Disconnected

### DISCUSSION THREAD

**Discussion by VSTLeonardo** [magma/magma#15313] [General]
Good afternoon everybody,

I am experiencing difficulties when trying to configure a femtocell on Magma. The MME (Evolved Packet Core) status remains "Disconnected" even after entering the femtocell's IP address in the NMS (Network Management System) settings. I would like to know if there are other settings that need to be performed in advance to resolve this problem.

Fento Data:

![image](https://github.com/magma/magma/assets/80913321/6e14b808-f9a1-45a1-add8-0328abe1184f)

**Comment by javieraubert** [comment]
Hello, to being able to connect you need to have the same TAC on enb and Magma, also the eNB must be pointing to the S1 interface of Magma, by default It is eth1. Also make sure that all services are up and running, you can use service magma@* status or the health_cli.py

**Comment by panyogesh** [comment]
Marking as answered and closing. If there are still issues raise a new thread.
